cub/cub/device/dispatch/dispatch_batch_memcpy.cuh

```diff
@@ -62,6 +62,12 @@
 
 CUB_NAMESPACE_BEGIN
 
+enum class MemcpyOption
+{
+  Memcpy,
+  Copy
+};
```

Comment:
Suggestion: I think `MemcpyOption::Memcpy` does not read nice. Maybe something like this would be better:
* `CopyImpl::Memcpy`
* `UseMemcpy::Yes`
* `CopyAlg::Memcpy`
* ...